exes/mqtt-broadcaster/tests/mqtt_broadcaster_tests.cpp

```diff
@@ -0,0 +1,294 @@
+#include <async_mqtt/all.hpp>
+#include <boost/ut.hpp>
+#include <tfc/ipc.hpp>
+#include "mqtt_broadcaster.hpp"
+
+namespace ut = boost::ut;
+using boost::ut::operator""_test;
+
+const int TIMEOUT_IN_MS = 100;
```

Comment:
I don't like to mix linguistics from C in C++ so I would recommend against using capital casing. 
```suggestion
constexpr int duration_100ms = 100;
```
It would be awesome if std::chrono::duration constructor would be constexpr ...

Reply:
Changed.